content/src/content/jcr_root/apps/core/wcm/components/list/v4/list/clientlibs/editor/js/list.js

```diff
@@ -0,0 +1,158 @@
+                var $target = $(event.target);
+                if ($target.is("foundation-autocomplete")) {
+                    updateLinkText($target);
+                    applyExternalLinksMode();
+                } else if ($target.is("coral-multifield")) {
+                    applyExternalLinksMode();
+                }
+            });
+
+            $staticMultifield.find("foundation-autocomplete input[is='coral-textfield']").on("blur", function() {
+                applyExternalLinksMode();
+            });
+
+            $("coral-select[name='./orderBy']").get(0).addEventListener("coral-select:showitems", function() {
+                applyExternalLinksMode();
+            });
+
+            handleListV3StaticPages($dialogContent, $staticMultifield);
+        }
+    });
+
+    $(document).on("foundation-contentloaded", function(e) {
+        $(".cmp-list__editor coral-select.cq-dialog-dropdown-showhide", e.target).each(function(i, element) {
+            var target = $(element).data("cqDialogDropdownShowhideTarget");
+            if (target) {
+                Coral.commons.ready(element, function(component) {
+                    showHide(component, target);
+                    applyExternalLinksMode();
+                    component.on("change", function() {
+                        showHide(component, target);
+                        applyExternalLinksMode();
+                    });
+                });
+            }
+        });
+        showHide($(".cq-dialog-dropdown-showhide", e.target));
+    });
+
+    function updateLinkText(target) {
+        var url = target.val();
+        if (url && url.startsWith("/")) {
+            var textField = target.parents("coral-multifield-item").find(staticLinkTextSelector);
+            if (textField) {
+                $.ajax({
+                    url: url + "/_jcr_content.json"
+                }).done(function(data) {
+                    if (data) {
+                        textField.val(data["jcr:title"]);
+                    }
+                });
+            }
+        }
+    }
+
+    function applyExternalLinksMode() {
+        var listFrom = $(".cmp-list__editor coral-select.cq-dialog-dropdown-showhide")[0].value;
+        var externals = $(staticMultifieldSelector).find("foundation-autocomplete").filter(function() {
+            var url = this.value;
+            return url && !url.startsWith("/");
+        });
+        if (externals.length === 0 || listFrom !== "static") {
+            $("coral-select-item[value='modified']").removeClass("hide");
+            $("coral-selectlist-item[value='modified']").removeClass("hide");
+            $("coral-tab:contains('Item Settings')").removeClass("hide");
+        } else {
+            $("coral-select-item[value='modified']").not(".hide").addClass("hide");
+            $("coral-selectlist-item[value='modified']").not(".hide").addClass("hide");
+            $("coral-tab:contains('Item Settings')").not(".hide").addClass("hide");
```

Comment:
That is supposed to work only when the static list has external links too, if the list has only internal pages then according to the requirements those options should be available.

@bpauli , I've tested it again and I haven't noticed any issue. Could you please add more details on how you reproduce the issue you noticed?

Reply:
Okay, now I understand the behaviour. So if I start a static list with internal list I can edit the properties and as soon I add an external item the properties settings disappear. To be honest, I thing this is over engineered and lead to some confusion for the authors. IMHO it would make more sense to just add a description in the properties tap which says that this settings don't have an effect on external link targets and that external links will not show one of this properties.